src/eventshub/src/component/SearchFilter/CustomRefinementList.tsx

```diff
@@ -29,6 +29,15 @@ const CustomRefinementList = ({ attribute, title, options }:CustomRefinementList
   const [isOpen, setIsOpen] = useState(false);
   const handleOpen = () => setIsOpen(!isOpen);
 
+  // Expand on mount if one of the options is selected.
+  useEffect(() => {
+    options.forEach((item) => {
+      if (selectedFilters.includes(item.name)) {
+        setIsOpen(true);
+      }
+    });
+  });
```

Comment:
```suggestion
 // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```
We need the dependency array to ensure the useEffect hook runs only once. Without it, the open accordion state persists 😃

Reply:
Ah, thanks for that fix. 🥂